client/js/test/table-view.test.js

```diff
@@ -11,6 +11,20 @@ describe('table view', () => {
 		document.documentElement.innerHTML = html;
 	});
 
+	describe('add row button', () => {
+		it('adds another row to the table body', () => {
```

Comment:
Make your test assertions describe the full functionality (like cause and effect). If I'm a dev who is less familiar with your code base, and I see that this test is failing, I should know that the button was supposed to "add another row to the table body WHEN CLICKED"